web-ide/proxy/static/css/webide.main.css

```diff
@@ -0,0 +1,101 @@
+  background-size: 56px !important;
+  width: 56px !important;
+  margin-right: 4px; 
+  margin-right: 4px; 
+}
+/**************************************************************************************************/
+
+/*************************** Hide menu items ******************************************************/
+.part.titlebar .menubar div.menubar-menu-button[aria-label="Terminal"],
+.part.titlebar .menubar div.menubar-menu-button[aria-label="Debug"]{
+  display: none !important;
+}
+/**************************************************************************************************/
+
+
+/*************************** Hide activity bar items **********************************************/
+.part.activitybar ul.actions-container li:nth-child(3),
```

Comment:
What do these correspond to? Do they have aria-labels? That's quite a nice way of documenting it.

Reply:
unfortunately, those aria-labels have system specific / special characters in them. Chose this more concrete selector instead of the hassle and unknown (I'll add a comment)